cpu/native/socket_zep/socket_zep.c

```diff
@@ -85,41 +68,41 @@ static inline size_t _zep_hdr_fill(socket_zep_t *dev, zep_hdr_t *hdr,
                                  payload_len);
 }
 
-static size_t _prep_vector(socket_zep_t *dev, const struct iovec *vector,
+static size_t _prep_vector(socket_zep_t *dev, const iolist_t *iolist,
                            unsigned n, struct iovec *out)
 {
-    size_t bytes = 0;
+    size_t bytes;
     dev->chksum_buf = 0;
 
-    for (unsigned i = 0; i < n; i++) {
-        bytes += vector[i].iov_len;
-    }
+    bytes = iolist_count(iolist);
```

Comment:
this should be `bytes = iolist_size(iolist);`

Reply:
fixed. thanks!